libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2793,6 +2793,43 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
       },
     ],
   },
+  xdcnetwork: {
+    type: "CryptoCurrency",
+    id: "XDC",
+    coinType: 550,
+    name: "XDCNetwork",
+    managerAppName: "XDCNetwork",
+    ticker: "XDC",
+    scheme: "xdc",
+    color: "#2b5876",
+    family: "ethereum",
+    ethereumLikeInfo: {
+      baseChain: "mainnet",
+      chainId: 50,
+      networkId: 50,
+      rpc: "https://rpc.xdcrpc.com",
+      chainId: 50,
+      explorer: {
+        uri: "https://xdc.blocksscan.io",
+        type: "blocksscan",
```

Comment:
At the moment only `etherscan` and `blockscout` APIs are supported. Is the `xdc.blocksscan.io` explorer compatible with one or the other API?

Reply:
Hello,

Yes BlocksScan Explorer's are compatible with EtherScan and BlocksScout API hence this would work. :)

Reply:
Let's use etherscan for now.